components/ClientRecordingPicker.js

```diff
@@ -31,6 +32,23 @@ const ClientRecordingPicker = () => {
     setIsModalVisible(false);
   };
 
+  // const renderRecordingButton = (buttonText = '') => {
```

Comment:
In general, you don't want to commit big commented out blocks of code. If you have experimental stuff that you don't want to use yet, it's a better idea to do something with `git stash`, or even a separate `experimental` branch that you merge into your own stuff. Let me know if you questions 🤙

Reply:
Cool, never used git stash - will certainly look into it for sure. But most def familiar w/ experimental branches.